content/en/apps/guides/hosting/4.x/self-hosting-multiple-nodes.md

```diff
@@ -191,85 +184,74 @@ Note that secure passwords and UUIDs were generated and saved in the resulting `
 
 #### CouchDB Node 1
 
-Create `/home/ubuntu/cht/docker-compose.yml` on Node 1.  It is the master node of the cluster and has different `environment:` settings than Node 2 and 3, note `CLUSTER_PEER_IPS` on line 15 for example:
+Create `/home/ubuntu/cht/docker-compose.yml` on Node 1 by running this code:
 
-{{< highlight yaml "linenos=table" >}}
+```shell
+cd /home/ubuntu/cht/
+curl -s -o ./docker-compose.yml https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:4.0.1/docker-compose/cht-couchdb.yml
+```
+
+Now create the override file to have Node 1 join the `cht-net` overlay network we created above. As well, we'll set some `services:` specific overrides:
+
+```shell
+cat > /home/ubuntu/cht/network-overrides.yml << EOF
```

Comment:
The file has more than just network overrides so maybe we should call it `custom-overrides.yml`?